src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -403,6 +409,180 @@ export const addForm0781 = formData => {
   return clonedData;
 };
 
+/**
+ * Extracts the month and year from a date string in the format YYYY-MM-DD
+ * If the date is missing or invalid, returns empty strings
+ *
+ * @param {string} dateString the date string to parse
+ * @returns {object} object containing `treatmentMonth` (MM) and `treatmentYear` (YYYY)
+ */
+export function extractDateParts(dateString) {
+  const match = dateString?.match(/^(\d{4})-(\d{2})/);
+  return {
+    treatmentMonth: match ? match[2] : '',
+    treatmentYear: match ? match[1] : '',
+  };
+}
```

Comment:
Should we be doing this?  Does the form specify it should be only month and year? I think we only ask for month and year on the frontend.... is it just like a date object even though they dont add day? IE if they enter `01/2022` since it is a date, not a string, it gets passed through to here as `01/01/2022`?

Reply:
Date(s) of treatment on the paper form are in the format MM-YYYY. In the case of a VA facility, month and year are optional, whereas month, day, and year are required for a private provider. Both are stored as strings, with a VA facility from date appearing as, e.g., "2025-03-XX", if both month and year are entered, and a private provider from date appearing as, e.g., "2025-03-10".